.github/workflows/m68k.yml

```
@@ -88,6 +91,19 @@ jobs:
         sudo mount debian-m68k.img vm
         sudo cp $(which qemu-m68k-static) vm/usr/bin/
 
+    - name: Prepare for build with target as JSON spec
+      run: |
+        ./y.sh prepare --only-libcore --cross
+
+    - name: Build sysroot with target as JSON spec
+      run: |
+        ./y.sh build --target-triple m68k-unknown-linux-gnu --target ${{ github.workspace }}/target_specs/m68k-unknown-linux-gnu.json
+
+    - name: Build sample project with target defined as JSON spec
+      run: |
+        ./y.sh cargo build --manifest-path=${{ github.workspace }}/tests/hello-world/Cargo.toml --target ${{ github.workspace }}/target_specs/m68k-unknown-linux-gnu.json
```

The following should work:
```suggestion
        ./y.sh cargo build --manifest-path=./tests/hello-world/Cargo.toml --target ${{ github.workspace }}/target_specs/m68k-unknown-linux-gnu.json
```